app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -0,0 +1,35 @@
+module Idv
+  class HowToVerifyController < ApplicationController
+    include RenderConditionConcern
```

Comment:
I imagine this is on your to-do already but: we should include guards so that if the user hasn't completed the previous page they get bumped back. And if the user already completed this page they get bumped forward